test/functional/llmq-is-cl-conflicts.py

```diff
@@ -106,19 +106,34 @@ def test_chainlock_overrides_islock(self, test_block_conflict):
 
         block = self.create_block(self.nodes[0], [rawtx2_obj])
         if test_block_conflict:
+            # The block shouldn't be accepted/connected but it should be known to node 0 now
             submit_result = self.nodes[0].submitblock(ToHex(block))
             assert(submit_result == "conflict-tx-lock")
 
         cl = self.create_chainlock(self.nodes[0].getblockcount() + 1, block.sha256)
         self.test_node.send_clsig(cl)
 
+        self.wait_for_best_chainlock(self.nodes[0], "%064x" % block.sha256)
+        bestchainlock = self.nodes[0].getbestchainlock()
+        assert(bestchainlock["blockhash"] == "%064x" % block.sha256)
+        assert(bestchainlock["known_block"] == test_block_conflict)
+
         self.wait_for_best_chainlock(self.nodes[1], "%064x" % block.sha256)
```

Comment:
shouldn't it be enough to replace this old call with a looped call for all nodes and then do a `sync_blocks`?

Reply:
Hmm.. yeah, I guess that would be clearer :) will fix